app/services/credit_notes/create_from_progressive_billing_invoice.rb

```diff
@@ -17,13 +17,13 @@ def call
       # Important to call this method as it modifies @amount if needed
       items = calculate_items!
 
-      CreditNotes::CreateService.new(
+      CreditNotes::CreateService.call(
```

Comment:
you can use `call!` it will raise if error.


```suggestion
      CreditNotes::CreateService.call!(
```